app/assets/javascripts/components/composition-form.jsx

```diff
@@ -41,12 +46,32 @@ export default class CompositionForm extends React.Component {
     this.setState({ players })
   }
 
-  onHeroSelection(hero, player) {
-    // TODO: actually save the selection of player + hero
+  onHeroSelectedForPlayer(heroID, player) {
+    const { composition } = this.state
+    const api = new OverwatchTeamCompsApi()
+
+    const body = {
+      hero_id: heroID,
+      map_id: composition.map.id,
+      player_name: player.name
+    }
+    if (composition.id) {
+      body.composition_id = composition.id
```

Comment:
Is it possible to set this without the `if` conditional? If there isn't a `composition.id`, won't the `body.composition_id` be null regardless?